server/container/core/src/main/java/org/apache/james/core/MailImpl.java

```diff
@@ -541,6 +542,7 @@ private void writeObject(java.io.ObjectOutputStream out) throws IOException {
         out.writeObject(remoteAddr);
         out.writeObject(lastUpdated);
         out.writeObject(attributes);
+        out.writeObject(perRecipientSpecificHeaders);
```

Comment:
Stupid question why here we don't just do `out.writeObject(this)` and annotate with `@transient` all the field we do not want ?